HipEmitLoweredNames fails to build

```
/home/pjaaskel/src/chip-spv/llvm_passes/HipEmitLoweredNames.cpp:46:8: error: no template named 'optional' in namespace 'std'; did you mean 'Optional'?
static std::optional<StringRef> getString(Constant *C) {
       ^~~~~~~~~~~~~
       Optional
/usr/lib/llvm-15/include/llvm/Transforms/Utils/LoopUtils.h:50:29: note: 'Optional' declared here
template <typename T> class Optional;
                            ^
/home/pjaaskel/src/chip-spv/llvm_passes/HipEmitLoweredNames.cpp:52:12: error: no member named 'nullopt' in namespace 'std'; did you mean 'nulls'?
    return std::nullopt;
           ^~~~~~~~~~~~
           nulls
/usr/lib/llvm-15/include/llvm/Support/raw_ostream.h:593:14: note: 'nulls' declared here
raw_ostream &nulls();
             ^
/home/pjaaskel/src/chip-spv/llvm_passes/HipEmitLoweredNames.cpp:52:17: error: no viable conversion from returned value of type 'llvm::raw_ostream &()' to function return type 'std::Optional<StringRef>'
    return std::nullopt;
....
```
clang++ v15. Some 'using' clause missing?

--- a/llvm_passes/HipEmitLoweredNames.cpp
+++ b/llvm_passes/HipEmitLoweredNames.cpp
@@ -34,6 +34,7 @@
 #include "llvm/Support/CommandLine.h"
 
 #include <fstream>
+#include <optional>
 
 #define PASS_NAME "emit-lowered-names"
 #define DEBUG_TYPE PASS_NAME

